src/settings/ui.ts

```diff
@@ -86,8 +87,11 @@ export function updateSuggestions(suggestions: string[], editor: Editor) {
             const beforeCursor = line.substring(0, cursor.ch);
             const match = beforeCursor.match(/(\b[\w']+)$/);
             if (match) {
+                if (settings.addSpace) {
+                    suggestion += " ";
+                }
```

Comment:
This is a bit cleaner in this case in my opinion. If you like bracketing better, make sure to add a whitespace before the following `editor.replaceRange();`.
```suggestion
                if (settings.addSpace) suggestion += " ";
```